tree/ntuple/v7/inc/ROOT/RPageStorageDaos.hxx

```diff
@@ -108,6 +108,7 @@ private:
    std::uint64_t fNBytesCurrentCluster{0};
 
    RDaosNTupleAnchor fNTupleAnchor;
+   bool fUseCaging = false;
```

Comment:
See long comment on use of "feature flags" and upper-bound of cage sizes.  Rationale: given that this feature might be useful for other storage backends, this might be better stored somewhere where else.

To be discussed, however.